mediascrapers.py

```diff
@@ -136,24 +142,30 @@ def scrape(self, url):
         media_urls = []
         soup = bs(source, 'html.parser')
         title = soup.find('title').text
-        for link in soup.find_all('a', href=True):
-            if is_media(link['href']):
-                media_urls.append(link['href'])
-            if is_media(link.text):
-                media_urls.append(link.text)
+        all_links=[]
```

Comment:
I think a better idea here, instead of commenting the backend code, would be to use that `strict` argument for `is_media`. 
Then pass `strict=True` for all `soup.find_all` except `soup.find_all('img')`

Reply:
oh makes sense yes.